src/index.js

```diff
@@ -60,7 +60,7 @@ class ProcessHtml {
         const ignoredFromPartial = ignoreLinksFromPartialMatches.filter(partial => href.indexOf(partial) >= 0);
 
         if (ignoreLinks.indexOf(href) < 0 && ignoredFromPartial.length === 0) {
-          source += `\nimport '${path}';\n`;
+          source += `\nimport '${path.replace(/\\/g, '\\\\')}';\n`;
```

Comment:
https://github.com/webpack-contrib/istanbul-instrumenter-loader/blob/master/test/utils/loader.js#L8

https://github.com/webpack/memory-fs/blob/master/lib/normalize.js

But not sure if it is the correct context in the code here :)